cmd/clair/main.go

```diff
@@ -53,129 +50,54 @@ func main() {
 		golog.Fatalf("failed to validate config: %v", err)
 	}
 
-	// setup global log level
-	var out io.Writer = os.Stderr
-	ll := logLevel(&conf)
-	if ll == zerolog.DebugLevel {
-		out = zerolog.ConsoleWriter{
-			Out: os.Stderr,
-		}
+	// initialize performs all Clair initialization tasks.
+	init, err := initialize.New(conf)
+	if err != nil {
+		golog.Fatalf("initialized failed: %v", err)
 	}
-	log := zerolog.New(out).With().
-		Timestamp().
-		Logger().
-		Level(ll)
+	logger := zerolog.Ctx(init.GlobalCTX).With().Str("component", "main").Logger()
 
-	// create global application context
```

Comment:
The `main` function should still create and control the global context.

Reply:
Why? A single ctx that drives the application globally is created. Theres no ambiguity.

Reply:
Why? A single ctx that drives the application globally is created. Theres no ambiguity.